.github/workflows/extended.yml

```diff
@@ -32,6 +32,8 @@ on:
   push:
     branches:
       - main
+      # release candidate branches
+      - 'branch-*'
```

Comment:
Same in docs & audit workflows?

Reply:
There are also `github.ref_name == 'main'` checks in the rust workflow. 
Currently they look harmless, as they only conditionally enable "some cache" (no sure what that cache is for...). 
Maybe worth adding a code comment in these usages to make it less likely someone will clopy-paste into a more important job step?